build-all.sh

```diff
@@ -65,28 +72,42 @@ if [ -e "$BUILDSTATUS_FILE" ]; then
 	echo "Continuing build-all from: $BUILDSTATUS_FILE"
 fi
 
-exec >	>(tee -a "$BUILDALL_DIR"/ALL.out)
-exec 2> >(tee -a "$BUILDALL_DIR"/ALL.err >&2)
-trap 'echo ERROR: See $BUILDALL_DIR/${PKG}.err' ERR
+exec &>	>(tee -a "$BUILDALL_DIR"/ALL.out)
+trap 'echo ERROR: See $BUILDALL_DIR/${PKG}.out' ERR
```

Comment:
This is still not mentioned in commit message, and I am still not convinced changing how logs are created are a good idea. I think it should be in a separate PR so that it can be properly reviewed, and anyone that might be using it can chime in.

Reply:
I can go either way on this.
It works well from my testing.
But I'm also not sure why this change is necessary.

It feels like this is probably better left for another PR.